build.go

```diff
@@ -63,8 +65,8 @@ type archiveFile struct {
 
 var targets = map[string]target{
 	"all": {
-		// Only valid for the "build" and "install" commands as it lacks all
-		// the archive creation stuff.
+		// Only valid for the "build", "install", "tar" and "zip" commands
```

Comment:
Actually, why not add the required stuff to this target, maybe keep the later magic to add all binaries automatically, instead of reusing the syncthing target? The filenames etc will make more sense that way.